scripts/configure-cluster/common/setup-operators.sh

```diff
@@ -36,10 +36,42 @@ install_service_binding_operator(){
 EOF
 }
 
+install_postgres_operator(){
```

Comment:
I think that this should be part of the tests, as the operator should be installed in the project of a specific test, not the same one for all tests.

Reply:
I have been thinking about this and thought that we can follow a convention where if someone wants to test non cluster wide operators they use `odo-operator-tests` as the namespace?

Reply:
Confirmed with @mohammedzee1000 that a user cannot install operators as part of the tests and that requires admin priv.

Reply:
You can switch between users inside the test scenario. You can easily login as a `cluster:admin` install operator, and then login back in as a normal user.

Reply:
done, now the cluster installation happens as expected.